src/api/js/src/high-level/high-level.ts

```diff
@@ -677,6 +700,44 @@ export function createApi(Z3: Z3Core): Z3HighLevel {
         );
       },
     };
+    const Array = {
+      sort<DomainSort extends AnySort<Name>[], RangeSort extends AnySort<Name>>(
+        ...sig: [...DomainSort, RangeSort]
+      ): SMTArraySort<DomainSort, RangeSort, Name> {
+        const arity = sig.length - 1;
+        const r = sig[arity];
+        const d = sig[0];
+        if (arity === 1) {
```

Comment:
Probably would help to have a useful error in the case that this is invoked with only a single argument (i.e. just a range, no domain). The type signature doesn't currently prevent that, I don't think.

Reply:
Good suggestion! I instead opted to now update the type signature to not allow it.